include/UIKit/UIButton.h

```diff
@@ -86,7 +86,7 @@ UIKIT_EXPORT_CLASS
 @property (nonatomic) UIEdgeInsets contentEdgeInsets;
 @property (nonatomic) UIEdgeInsets imageEdgeInsets;
 @property (nonatomic) UIEdgeInsets titleEdgeInsets;
-@property (nonatomic, readonly) UIButtonType buttonType;
+@property (nonatomic, readonly) UIButtonType buttonType STUB_PROPERTY;
```

Comment:
To be honest, I can remove this change. Jared has a PR out that adds this feature. I'll remove.